### PR TITLE
Change warning in debug mode

### LINKED ISSUE #3793: Warning says to enable debug when debug is already enabled
When running coala with `-V` , a warning message says to enable -V

```
Bear CheckstyleBear failed to run. Take a look at debug messages (`-V`) for further information.
```
https://ci.appveyor.com/project/coala/coala-bears/build/job/c9ca1wygut9y7eds

### PR DESCRIPTION
**Pull request by sinwar**
Fixes #3793

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: coalib/bears/Bear.py [hunk 1]
```diff
@@ -251,13 +251,15 @@ def execute(self, *args, **kwargs):
             return [] if result is None else list(result)
         except (Exception, SystemExit):
             if self.kind() == BEAR_KIND.LOCAL:
-                self.warn('Bear {} failed to run on file {}. Take a look '
-                          'at debug messages (`-V`) for further '
-                          'information.'.format(name, args[0]))
+                if '-V' not in args:
```
Comment (Udayan12167) [inline coalib/bears/Bear.py]: Better way is `self.log_level() == logging.DEBUG`

File: coalib/bears/Bear.py [hunk 2]
```diff
@@ -251,13 +251,15 @@ def execute(self, *args, **kwargs):
...
+                    self.warn('Bear {} failed to run on file {}. Take a look '
+                              'at debug messages (`-V`) for further '
+                              'information.'.format(name, args[0]))
             else:
-                self.warn('Bear {} failed to run. Take a look '
-                          'at debug messages (`-V`) for further '
-                          'information.'.format(name))
+                if '-V' not in args:
```
Comment (Udayan12167) [inline coalib/bears/Bear.py]: same here

File: coalib/bears/Bear.py (hunk 1 above)
Comment (sinwar) [inline coalib/bears/Bear.py]: But there is no property named log_level in bear.

File: coalib/bears/Bear.py (hunk 2 above)
Comment (sinwar) [inline coalib/bears/Bear.py]: using ```self.log_level()``` shows error-
There is not attribute named log_level in LocalBear or GlobalBear

**Comment by meetmangukiya** [comment]
@sinwar does this mean you want to unassign yourself from the issue?

**Pull request closed without merging**